cardano-cli/src/Cardano/CLI/Shelley/Run/Pool.hs

```diff
@@ -158,8 +156,8 @@ runStakePoolRetirementCert (VerificationKeyFile sPvkeyFp) retireEpoch (OutputFil
       . newExceptT
       $ readFileTextEnvelope (AsVerificationKey AsStakePoolKey) sPvkeyFp
 
-    let stakePoolId = verificationKeyHash stakePoolVerKey
-        retireCert = makeStakePoolRetirementCertificate stakePoolId retireEpoch
+    let stakePoolId' = verificationKeyHash stakePoolVerKey
+        retireCert = makeStakePoolRetirementCertificate stakePoolId' retireEpoch
```

Comment:
Did `stakePoolId` have to be renamed here?

Reply:
Yep.  This was the error:

```
src/Cardano/CLI/Shelley/Run/Pool.hs:159:9: error: [-Wname-shadowing, -Werror=name-shadowing]
    This binding for ‘stakePoolId’ shadows the existing binding
      imported from ‘Cardano.Api.Typed’ at src/Cardano/CLI/Shelley/Run/Pool.hs:19:1-34
    |
159 |     let stakePoolId = verificationKeyHash stakePoolVerKey
    |         ^^^^^^^^^^^
```